### DISCUSSION TITLE
Major classification axes of SeREEn

### DISCUSSION THREAD

**Discussion by pbuttigieg** [SeREEnFederation/SeREEn#3] [Ideas]
SeREEn wants to be inclusive to all forms of semantic artifacts and resources, while being clear about what they are and how they should be used. Thus, we'll think about how to classify registered entries along a few axes.

Those below were briefly discussed during our ESIP call today.

# Major categories of semantic resources in our registry

1. Machine-actionability (TRL thinking)
   1. Operational resources
   1. Level of Expressivity…
1. Maintenance
   1. Project
   1. Programme (many projects)
   1. Community across many programmes (long-term)
   1. …
1. Type of resources
   1. Ontology graphs (DAGs)
   1. Look-up services
   1. Thesauri
   1. Knowledge Graphs… 
   1. UML diagrams
   1. Textual descriptions of semantic resources or proposed ontologies

**Comment by semantic-nomad** [comment]
Perhaps also linkage --- what does the structure import, extend, or reuse?

**Comment by rduerr** [comment]
Looking at [MOD2](https://www.isibang.ac.in/ns/mod/index.html) (ontology is at https://github.com/sifrproject/MOD-Ontology) 
and [DCAT](https://w3c.github.io/dxwg/dcat/) for possible source of metadata for describing these things....

**Comment by dr-shorthair** [comment]
Since a 'semantic resource' can be seen as a special case of 'dataset', I would see MOD as a specialization or 'profile' of DCAT.

**Comment by semantic-nomad** [comment]
Yes, MOD as a DCAT profile appears to be the proposal, as [discussed at the most recent FAIRsFAIR workshop](https://drive.google.com/file/d/1iXWLr71-QfZQFWuzkQ6hsG_vu2u1ZeKF/view).

**Comment by rduerr** [comment]
Is this a good example of 3.ii.?

  https://lov.linkeddata.es

**Comment by rduerr** [comment]
where would https://wordnik.com/ fit?
